application.go

```diff
@@ -34,6 +34,25 @@ type Applications struct {
 	Apps []Application `json:"apps"`
 }
 
+// IpAddress is used by Ip Per Task Funcionality https://mesosphere.github.io/marathon/docs/ip-per-task.html
```

Comment:
Good point about the name clash.

I'm still trying to make sense of out of Marathon's documentation. Do I understand correctly that the struct must be set/populated in order for the IP-per-task feature to be activated? If so, then I'd probably go with `IPAddressPerTask` as the name.